### PR TITLE
Edited the section on command-line quickstart

### PR DESCRIPTION
**Pull request by walkerlj0**
Added a couple lines about the UI/ Web console to tell how to view your ipfs file with it

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/how-to/command-line-quick-start.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://github.com/ipfs-shipyard/ipfs-desktop
- The following link returned HTTP status code 301 (Moved Permanently):
  https://github.com/ipfs-shipyard/ipfs-companion
- The following link returned HTTP status code 301 (Moved Permanently):
  https://addons.mozilla.org/firefox/addon/ipfs-companion/
- The following link returned HTTP status code 301 (Moved Permanently):
  https://golang.org/doc/install
- Possible missing comma found.
  ```
  ...reference implementation written in Go. TIP Don’t want to use the command li...
                                             ^^^
  ```
- The word ‘install’ is not a noun.
  ```
  ...e not yet installed Go-IPFS, follow the install instructions. # Initialize t...
                                             ^^^^^^^
  ```
- The phrase ‘a lot of’ might be wordy. Consider using “many”, “numerous” or “countless”.
  ```
  ...oing so will prevent IPFS from creating a lot of data center-internal traffi...
                                             ^^^^^^^^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...nd and connect to you. You can run ipfs id at any time to get it again if yo...
                                             ^^
  ```
- This word has been used in one of the immediately preceding sentences. Using a synonym could make your text more interesting to read, unless the repetition is intentional.
  ```
  ...s id at any time to get it again if you need it. Now, try running the comman...
                                             ^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... your computer. The gateway queried the Distributed hash table (DHT), found ...
                                             ^^^^^^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...way you would a name-based file system. in order to get ipfs files to show u...
                                             ^^^^^^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...led You need to install and set up FUSE in order to mount the file system. F...
                                             ^^^^^^^^^^^
  ```

File: docs/how-to/command-line-quick-start.md [hunk 1]
```diff
@@ -148,6 +148,14 @@ You can view the web console on your local node by going to `localhost:5001/webu
 
 ![Web console connection view](./images/command-line-quick-start/webui-connection.png)
 
+The web console shows files that are in your [Mutable File System (MFS)](https://docs.ipfs.io/concepts/file-systems/#mutable-file-system-mfs) which is a tool built into the web console that helps you navigate IPFS files in the same way you would a name-based file system.
```
Comment (johnnymatthews) [inline docs/how-to/command-line-quick-start.md:151]:
```suggestion
The web console shows files that are in your [Mutable File System (MFS)](../concepts/file-systems#mutable-file-system-mfs). MFS is a tool built into the web console that helps you navigate IPFS files in the same way you would a name-based file system.
```

Links need to be relative to the rest of the repo.

File: docs/how-to/command-line-quick-start.md [hunk 2]
```diff
@@ -148,6 +148,14 @@ You can view the web console on your local node by going to `localhost:5001/webu
...
+
+in order to get ipfs files to show up in the web console (including ones you [added using the CLI](https://docs.ipfs.io/reference/cli/#ipfs-add)), copy them over to MFS:
```
Comment (johnnymatthews) [inline docs/how-to/command-line-quick-start.md:153]:
```suggestion
When you add files using the [CLI command `ipfs add ...`](../cli#ipfs-add), these files are not automatically available within the MFS. To view files in IPFS Desktop that you added using the CLI, you must copy the files over to the MFS:
```

File: docs/how-to/command-line-quick-start.md [hunk 3]
```diff
@@ -148,6 +148,14 @@ You can view the web console on your local node by going to `localhost:5001/webu
...
+
+```
+ipfs files cp /ipfs/<ipfs-CID>
+```
```
Comment (johnnymatthews) [inline docs/how-to/command-line-quick-start.md:157]:
````suggestion
```shell
ipfs files cp /ipfs/<ipfs-CID>
```
````

Code blocks must have a language assigned to them. If there isn't a relevant language, use ```plaintext.

**Review by johnnymatthews — CHANGES_REQUESTED** [review]
Made a couple of suggestions, mainly around links. Take a look and let me know if you've got questions.

File: docs/how-to/command-line-quick-start.md (hunk 2 above)
Comment (walkerlj0) [reply docs/how-to/command-line-quick-start.md:153]: I think it's worth having some kind of context here for MFS-- let users know within this that MFS is the thing that lets people know it is what the IPFS uses and that it is separate from the rest of the IPFS docs

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/how-to/command-line-quick-start.md`:
- Possible missing comma found.
  ```
  ...reference implementation written in Go. TIP Don’t want to use the command li...
                                             ^^^
  ```
- The word ‘install’ is not a noun.
  ```
  ...e not yet installed Go-IPFS, follow the install instructions. # Initialize t...
                                             ^^^^^^^
  ```
- The phrase ‘a lot of’ might be wordy. Consider using “many”, “numerous” or “countless”.
  ```
  ...oing so will prevent IPFS from creating a lot of data center-internal traffi...
                                             ^^^^^^^^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...nd and connect to you. You can run ipfs id at any time to get it again if yo...
                                             ^^
  ```
- This word has been used in one of the immediately preceding sentences. Using a synonym could make your text more interesting to read, unless the repetition is intentional.
  ```
  ...s id at any time to get it again if you need it. Now, try running the comman...
                                             ^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... your computer. The gateway queried the Distributed hash table (DHT), found ...
                                             ^^^^^^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...way you would a name-based file system. in order to get ipfs files to show u...
                                             ^^^^^^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...led You need to install and set up FUSE in order to mount the file system. F...
                                             ^^^^^^^^^^^
  ```

**Review by walkerlj0 — COMMENTED** [review]
Add something that lets people know 'MFS is something separate that you need to add ipfs files to, in order to view in the CLI'

File: docs/how-to/command-line-quick-start.md (hunk 2 above)
Comment (walkerlj0) [inline docs/how-to/command-line-quick-start.md]:
```suggestion
in order to get ipfs files to show up in the web console (including ones you [added using the CLI](https://docs.ipfs.io/reference/cli/#ipfs-add)), copy them over to MFS, which is a tool built into the IPFS client used for navigating IPFS files:
```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/how-to/command-line-quick-start.md`:
- The word ‘install’ is not a noun.
  ```
  ...e not yet installed Go-IPFS, follow the install instructions. # Initialize t...
                                             ^^^^^^^
  ```
- The phrase ‘a lot of’ might be wordy. Consider using “many”, “numerous” or “countless”.
  ```
  ...oing so will prevent IPFS from creating a lot of data center-internal traffi...
                                             ^^^^^^^^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...nd and connect to you. You can run ipfs id at any time to get it again if yo...
                                             ^^
  ```
- This word has been used in one of the immediately preceding sentences. Using a synonym could make your text more interesting to read, unless the repetition is intentional.
  ```
  ...s id at any time to get it again if you need it. Now, try running the comman...
                                             ^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... your computer. The gateway queried the Distributed hash table (DHT), found ...
                                             ^^^^^^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...led You need to install and set up FUSE in order to mount the file system. F...
                                             ^^^^^^^^^^^
  ```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/how-to/command-line-quick-start.md`:
- The word ‘install’ is not a noun.
  ```
  ...e not yet installed Go-IPFS, follow the install instructions. # Initialize t...
                                             ^^^^^^^
  ```
- The phrase ‘a lot of’ might be wordy. Consider using “many”, “numerous” or “countless”.
  ```
  ...oing so will prevent IPFS from creating a lot of data center-internal traffi...
                                             ^^^^^^^^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...nd and connect to you. You can run ipfs id at any time to get it again if yo...
                                             ^^
  ```
- This word has been used in one of the immediately preceding sentences. Using a synonym could make your text more interesting to read, unless the repetition is intentional.
  ```
  ...s id at any time to get it again if you need it. Now, try running the comman...
                                             ^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... your computer. The gateway queried the Distributed hash table (DHT), found ...
                                             ^^^^^^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...led You need to install and set up FUSE in order to mount the file system. F...
                                             ^^^^^^^^^^^
  ```

**Review by johnnymatthews — APPROVED** [review]

**Comment by johnnymatthews** [comment]
Looks good! Merged!

**Pull request merged**